e2e/support/helpers/e2e-embedding-helpers.js

```diff
@@ -182,7 +182,7 @@ export function getIframeBody(selector = "iframe") {
     .its("0.contentDocument")
     .should("exist")
     .its("body")
-    .should("not.be.null")
+    .should("not.be.undefined")
```

Comment:
I found out sometimes `not.be.null` would fail. Using `undefined` follows [Cypress' guide](https://www.cypress.io/blog/working-with-iframes-in-cypress) here